draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -317,7 +317,7 @@ The payload of an SD-JWT is a JSON object according to the following rules:
  1. The payload MAY contain the `_sd_alg` key described in (#hash_function_claim).
  2. The payload MAY contain one or more digests of Disclosures to enable selective disclosure of the respective claims, created and formatted as described in (#creating_disclosures).
  3. The payload MAY contain one or more decoy digests to obscure the actual number of claims in the SD-JWT, created and formatted as described in (#decoy_digests).
- 4. The payload MAY contain one or more non-selectively disclosable claims.
+ 4. The payload MAY contain one or more permanently disclosable claims.
```

Comment:
```suggestion
 4. The payload MAY contain one or more permanently disclosed claims.
```

Reply:
This should be changed everywhere (search + replace?)

Reply:
yeah, you are correct. i'll fix